binja/main.py

```diff
@@ -0,0 +1,62 @@
+import os
+import subprocess
+import sys
+from shutil import copy
+import tarfile
+import json
+
+files_set = set()
+curdir = os.getcwd()
+
+# Handle an individual file
+def analyze(exe, core_file):
+    global files_set
+    pygdb = curdir + "/gdb.py"
+    lines = (
+        subprocess.check_output(
+            ["gdb", exe, core_file, "-batch", "-ex", "source " + pygdb, "--quiet"]
+        )
+        .decode("utf-8")
+        .split("\n")[4:]
+    )
+
+    files = []
+    for l in lines:
```

Comment:
The parsing in this loop is a little opaque &mdash; maybe add a short description of the line format it's looking for at the top and/or use regexes here?